packages/zimbabwe/README.md

```diff
@@ -0,0 +1,37 @@
+# 🇿🇼 Zimbabwe
```

Comment:
Could you also create a `package.json` file like the other maps please? I forgot to write it in the [contribution guide](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md) 🙇

Reply:
I updated the contribution guide about the [package.json file](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md#package-file)